flink-cdc-composer/src/test/java/org/apache/flink/cdc/composer/flink/FlinkPipelineTransformITCase.java

```diff
@@ -2184,6 +2184,284 @@ void testTransformWithLargeLiterals() throws Exception {
                         "DataChangeEvent{tableId=default_namespace.default_schema.mytable1, before=[12th, 15, Oops, 2147483647, 2147483648, -2147483648, -2147483649, 1234567890123456789], after=[], op=DELETE, meta=()}");
     }
 
+    @Test
+    void testFloorCeilAndRoundFunctionWithLiterals() throws Exception {
```

Comment:
Seems these cases aren't testing calling function with literals (but with specific column values).

```suggestion
    void testFloorCeilAndRoundFunctions() throws Exception {
```